package.json

```diff
@@ -504,83 +510,102 @@
-          "type": "string",
-          "default": "latest",
-          "markdownDescription": "Docker image tag to use when starting a local Schema Registry container. (By default, this will use the [`confluentinc/cp-schema-registry`](https://hub.docker.com/r/confluentinc/cp-schema-registry/tags) image.)"
-        },
-        "confluent.topic.produceMessages.schemas.useTopicNameStrategy": {
-          "type": "boolean",
-          "default": true,
-          "markdownDescription": "Use [TopicNameStrategy](https://docs.confluent.io/platform/current/schema-registry/fundamentals/serdes-develop/index.html#overview) for schema subject name when producing messages to Kafka topics. Disabling this will require selecting the subject name strategy to use."
-        },
-        "confluent.topic.produceMessages.schemas.allowOlderVersions": {
-          "type": "boolean",
-          "default": false,
-          "markdownDescription": "Allow older schema versions to be used when producing messages to Kafka topics.\n\n- Enabling this will require selecting the schema version to use.\n\n- Disabling this will use the **latest schema version**.\n\n---\n\n⚠️ **WARNING**: Producing messages with older schema versions can be in violation of some schema compatibility settings."
-        },
-        "confluent.preview.enableFlink": {
-          "type": "boolean",
-          "default": false,
-          "markdownDescription": "Enable Flink-related resource loading and associated actions.",
-          "tags": [
-            "preview"
-          ]
-        },
-        "confluent.experimental.enableChatParticipant": {
-          "type": "boolean",
-          "default": false,
-          "description": "Enable the Confluent chat participant for Copilot.",
-          "tags": [
-            "experimental"
-          ]
+          "confluent.experimental.enableChatParticipant": {
+            "type": "boolean",
+            "default": false,
+            "description": "Enable the Confluent chat participant for Copilot.",
+            "tags": [
+              "experimental"
+            ]
+          }
+        }
+      },
+      {
+        "title": "Flink SQL Settings",
+        "properties": {
+          "confluent.preview.enableFlink": {
+            "type": "boolean",
+            "default": false,
+            "order": 0,
+            "description": "Enable Flink-related resource loading and associated actions.",
+            "tags": [
+              "preview"
+            ]
+          },
+          "confluent.flink.computePoolId": {
+            "type": "string",
+            "default": "",
+            "order": 1,
+            "description": "ID for the default CCloud compute pool to use for Flink SQL operations."
+          },
+          "confluent.flink.database": {
+            "type": "string",
+            "default": "",
+            "description": "Name or ID for default CCloud database (topic) to use for Flink SQL operations"
```

Comment:
```suggestion
            "description": "Name or ID for default CCloud database (Kafka cluster) to use for Flink SQL operations"
```

Reply:
A topic is a Flink _table_. A Kafka cluster is a Flink _database_. All databases (clusters) in the org + cloud provider + cloud region make up the flink 'catalog', the collection of possibly queryable databases.

Reply:
Text from CCloud UI:

These are the default catalog and database that will be used to interpret any table references in your SQL. In Confluent Cloud, an environment is mapped to a Flink catalog, and a Kafka cluster is mapped to a Flink database.
When default catalog and/or database are not selected here, you must fully qualify table references in your SQL, like so: SELECT ... FROM my_catalog.my_database.my_table.